spec/controllers/movies_controller_spec.rb

```diff
@@ -36,19 +36,26 @@
       end
 
       it 'strips full youtube url' do
-        youtube_id = '44urisjfk'
         patch :update,
               { id: movie.id,
                 tmdb_id: movie.tmdb_id,
                 trailer: "https://www.youtube.com/watch?v=#{youtube_id}" }
         expect(movie.reload.trailer).to eq(youtube_id)
       end
 
+      it 'strips youtube url with additional params' do
+        patch :update,
+              { id: movie.id,
+                tmdb_id: movie.tmdb_id,
+                trailer: "https://www.youtube.com/watch?v=#{youtube_id}&ab_channel=A24" }
+        expect(movie.reload.trailer).to eq(youtube_id)
+      end
+
```

Comment:
We may want to add a couple of more cases in here to cover some of the logic in these private methods:
```suggestion
it 'leaves non-youtube urls as-is' do
        original_trailer = 'https://www.example.com'
        patch :update,
              { id: movie.id,
                tmdb_id: movie.tmdb_id,
                trailer: original_trailer }
        expect(movie.reload.trailer).to eq(original_trailer)
      end

      it 'handles empty trailer urls' do
        trailer = ''
        patch :update,
              { id: movie.id,
                tmdb_id: movie.tmdb_id,
                trailer: trailer }
        expect(movie.reload.trailer).to eq(trailer)
      end
```

Reply:
🤔 I did add those, but it's a bit weird because that's not how we _want_ it to work. 

But since it is how it works, I agree it'd be helpful to have the tests there. 

I put them in a context and added a comment referencing the other issue that aims to fix it.